app/room.py

```diff
@@ -0,0 +1,116 @@
+import random
+from os import sys, path
+
+from app.fellow import Fellow
+from app.staff import Staff
+from app.person import Person
+
+sys.path.append(path.dirname(path.dirname(
+    path.dirname(path.abspath(__file__)))))
+
+
+class Room():
+
+    total_number_of_rooms = 0
+    room_list = {}
+
+    def __init__(self, room_name):
+        self.name = room_name.capitalize()
```

Comment:
I decided to capitalize it for better display format when outputting a message to the user. I decided to capitalize it when invoking the `init()` so as to eliminate repetitive call to `capitalize()` whenever i want to output a message to the user.